ci/docker/Dockerfile

```diff
@@ -54,14 +55,21 @@ RUN . /etc/os-release && \
         'libnvparsers7=7.2.*+cuda11.0' \
         libtool \
         protobuf-compiler \
+        python3-dev \
         python3-pip \
         python3-six \
         python3-wheel && \
     apt-get clean && \
     find /var/lib/apt/lists -delete
 
+RUN python3 -m pip install -U pip setuptools wheel
+
 RUN apt-mark hold libcudnn8 libcudnn8-dev libnvinfer7 libnvinfer-dev libnvonnxparsers7 libnvonnxparsers-dev
 
+ENV PATH /usr/local/nvidia/bin:/usr/local/cuda-11.0/bin:${PATH}
+ENV LD_LIBRARY_PATH=/usr/local/cuda-11.0/targets/x86_64-linux/lib
+ENV CPLUS_INCLUDE_PATH=/usr/include/python3.6m
```

Comment:
`ENV LD_LIBRARY_PATH=/usr/local/cuda-11.0/targets/x86_64-linux/lib:${LD_LIBRARY_PATH}`
`ENV CPLUS_INCLUDE_PATH=/usr/include/python3.6m:${CPLUS_INCLUDE_PATH}`

And why `CPLUS_INCLUDE_PATH` is need to be changed?